washingtontrails/vms
# Features/personalized awards

This pull request will meet the requirements for user story 203 in scrumdo (Collect Award Personalization Info)

To Test:

Ensure that you have at least two awards listed in Salesforce (I've created three, but you may wish to experiment).  Each award should have an 'extra_info_needed' value set to either 'Nickname' or 'Size' and a 'collect personalization on day x' value of a positive integer (I've set some up).

Register for a work party as a user with enough incentive days to meet or exceed one of the above award 'collect personalization' values.  Earned incentive days will be combined with incentive days available from work parties for which the user is registered, so long as the status of that registration is either 'Registered' or 'Payment Pending' and so long long as the work party for that registration is still 'Open'. In addition, incentive days for the work party for which you are registering will be counted.

If the number of incentive days calculated match or exceed the value of the 'collect personalization on day x' value for an award, and if that award personalization has not yet been collected, then an extra field should appear on the registration form to require the user to provide that information.  

1.  Verify that the correct field appears for a given number of calculated incentive days
2.  Verify that if the customization has already been provided, the field is *not* shown on the form
3.  Verify that the submitted value for the customization is recorded correctly in SalesForce
